### PR TITLE
Feature 

### PR DESCRIPTION
**Pull request by Vishal30M**
Hello Microverse Project
I have finished the project as required.  I have applied the features as following:
Follow GitHub flow.
Added a .gitignore file.
Added node modules to a .gitignore file.
Setup linters for Html & CSS.
Added a descriptive README.md
Added 3 files HTML, CSS & test. md file each with a separate commit.
Added test.md to .gitignore

### CODE REVIEW

File: .github/workflows/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello - Microverse</title>
+</head>
+<body>
+    <header class="heading-1">
+        <h1>Hello-Microverse</h1>
+    </header>
+</body>
+</html>
```
Comment (Shedrack-Sunday) [inline .github/workflows/index.html:14]: You put in the effort to create this, however, you placed it in the wrong directory. Kindly read the Project requirements properly, and check out the guidelines on proper [Gitflow and linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided in the project requirements. Only the `linters.yml` file should be inside of the `.github/workflow folder. Please check the link I attached, and let's fix it to complement your good effort so far 👍

File: .github/workflows/style.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.heading-1{
+    color: blueviolet;
+}
```
Comment (Shedrack-Sunday) [inline .github/workflows/style.css:9]: As with the `index.html`, please check the proper setting of the [linter setup](https://github.com/microverseinc/linters-config/tree/master/html-css). Your `style.css` should be in the root of your directory or your preferred folder, but not in the `.github/workflows` folder. lets fix this and make it good 😃

File: README.md [hunk 3]
```diff
@@ -1,97 +1,26 @@
...
-If you are not familiar with linters, read [root level README](../README.md).
+# First project-hello-microverse
+In this project, I set up a "Hello Microverse" repository. No complex coding was required for this exercise. My goal was to master all of the tools and best practices I learned about in previous steps. 
 
-## Set-up GitHub Actions
+## Built With
 
-Please do the following **steps in this order**:
+- HTML
+- CSS
```
Comment (Shedrack-Sunday) [inline README.md:10]: You obviously put in effort on your README setup 👏 , however, you need to stick to the [README FILE](https://github.com/microverseinc/readme-template) provided with every section filled. The reason for this Is that when your future project will be viewed by a person of interest, let's say a recruiter, the first point of impression is the setup of your [README](https://github.com/microverseinc/readme-template). As a potential Software developer, you wouldn't want to give room for a bad impression 😃 . Kindly follow the link attached, and fill in your README proper 👏 

## Expected header:
jjjjjjjjjjjjjjj

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Vishal30M , 

You did a good job putting in some effort so far! 👏 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Changes Required ♻️ 



## Required Changes ♻️
- [ ] Your pull request seems to be empty. please fill it up explaining the changes you made in your project.
Empty pull request:
emtyyyy


- [x] Kindly make sure to fill up the READE template with all the features provided. I've listed the items omitted. It will make your work look  professional and well  documented
- [x] Based on Project requirements, you're meant to make three commit messages, though you made two, you didn't follow the direction which was to commit `html` `css` and the last `test.md` files. . 
- [ ] Based on project requirements, you were meant to add the `test.md` file in the `.gitignore` file, but it seems you forgot that. no worries, please fix that.

-  [ ] I noticed you didn't arrange the workflow properly. I've attached a screenshot of the project requirements and made a more detailed review in the comments. kindly check them, and make the required changes. good job with your efforts s far 👏 

rewwwww

- [x] Check below to see a more detailed review of what is needed. good job 👏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NahnahAJ — APPROVED** [review]
Hi @Vishal30M , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

:heavy_check_mark: Good commit messages
:heavy_check_mark: Good use of GitHub flow
:heavy_check_mark: Linters are correctly setup and passing
:heavy_check_mark: Descriptive PR title and description
:heavy_check_mark: Project requirements fulfilled

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**